### PR TITLE
Implement email confirmation API calls using ember-api-actions

### PR DESCRIPTION
**Pull request by Turbo87**
r? @locks

### CODE REVIEW

File: tests/acceptance/email-change-test.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ module('Acceptance | Email Change', function (hooks) {
 
       await click('[data-test-email-input] [data-test-resend-button]');
       assert.dom('[data-test-email-input] [data-test-resend-button]').isEnabled().hasText('Resend');
-      assert.dom('[data-test-email-input] [data-test-error]').hasText('Unknown error in resending message');
+      assert.dom('[data-test-email-input] [data-test-error]').hasText('Error in resending message: [object Object]');
```
Comment (Turbo87) [inline tests/acceptance/email-change-test.js:188]: This is not exactly pretty, but I didn't touch the formatting code, I just ensured that we actually return an Ember Data error from the method call. I have another PR queued up already that fixes the formatting and error display.

File: app/models/user.js [hunk 2]
```diff
@@ -13,4 +13,28 @@ export default class User extends Model {
...
+      user: {
+        id: this.id,
+        email,
+        email_verified: false,
+        email_verification_sent: true,
+      },
+    });
+  }
+
+  #changeEmail = memberAction({
```
Comment (locks) [inline app/models/user.js:29]: fancy!

Comment (Turbo87) [reply app/models/user.js:29]: it worked surprisingly well 😅

**Pull request merged**